### DISCUSSION TITLE
How to install SSL certificate with new Postal smtp ?

### DISCUSSION THREAD

**Discussion by fav22** [postalserver/postal#1746] [Help with using Postal]
Hi I hope you're fine ! I would like to ask how to install an ssl certificate with postal smtp, I tried with Caddy's line of code but it doesn't change anything, same with Ngix's. The problem is that the ssl is only valid on the main  web adress of my domain https://postal.piikla.com and not for http://postal.piikla.com:5000 when i add it with Ngix

**Comment by willpower232** [comment]
You should only access https://postal.piikla.com and not http://postal.piikla.com:5000, port 5000 should not be exposed at all.

It looks like you are currently using apache on that server so it might not be set up how you think.

**Reply by fav22** [reply]
Thank you for your answer, I have correctly configured postal smtp for the address https://postal.piikla.com when installing postal but I was forced to create a port 5000 in `postal.yml` and since then it is configured with https://postal.piikla.com:5000 only
